From 1903fa3fba7ae8138111f94d5cef023f5a050c2f Mon Sep 17 00:00:00 2001
From: Balamurugan Selvarajan <quic_bselvara@quicinc.com>
Date: Fri, 9 Sep 2022 13:32:45 +0530
Subject: [PATCH] ath12k: Add PPE config definition to enable/disable PPE
 support

Added PPE macro definition to enable/disable PPE support.

Signed-off-by: Balamurugan Selvarajan <quic_bselvara@quicinc.com>
---
 local-symbols        | 1 +
 net/mac80211/Kconfig | 9 +++++++++
 2 files changed, 10 insertions(+)

diff --git a/local-symbols b/local-symbols
index 945083a..1e65461 100644
--- a/local-symbols
+++ b/local-symbols
@@ -66,6 +66,7 @@ MAC80211_TDLS_DEBUG=
 MAC80211_DEBUG_COUNTERS=
 MAC80211_STA_HASH_MAX_SIZE=
 MAC80211_NSS_SUPPORT=
+MAC80211_PPE_SUPPORT=
 WLAN=
 PCMCIA_RAYCS=
 PCMCIA_WL3501=
diff --git a/net/mac80211/Kconfig b/net/mac80211/Kconfig
index 727e48c..2599669 100644
--- a/net/mac80211/Kconfig
+++ b/net/mac80211/Kconfig
@@ -113,6 +113,15 @@ config MAC80211_NSS_SUPPORT
 
 	  If unsure, say Y to enable NSS offload support.
 
+config MAC80211_PPE_SUPPORT
+	bool "QTI mac80211 ppe support"
+	depends on ATH12K
+	default n
+	---help---
+	  Enables PPE support for ATH12K driver
+
+	  If unsure, say N.
+
 config MAC80211_NOINLINE
 	bool "Do not inline TX/RX handlers"
 	depends on MAC80211_DEBUG_MENU
-- 
2.17.1

